docs/sphinx/examples/python/building_kernels.py

```diff
@@ -0,0 +1,266 @@
+# # Building Kernels
```

Comment:
Do we have a C++ equivalent? Examples are basic concepts that should be shown in both python and c++

Reply:
If it didnt already exist, we dont have C++ versions for it. Good to keep on the agenda for the next round of development.